.ci/pipeline_definitions

```diff
@@ -48,6 +48,12 @@ terraformer:
             target: terraformer
             build_args:
               PROVIDER: equinixmetal
+          terraformer-none:
+            registry: 'gcr-readwrite'
+            image: 'eu.gcr.io/gardener-project/gardener/terraformer-none'
```

Comment:
Don't know how to feel about `none`...
WDYT about `slim` or `standalone` or something similar indicating that there are no additional things installed in the image?
Naming is always hard. Don't have a clear preference here.

Reply:
I like `slim`, will change it to that unless someone objects.